electron/legendary/user.ts

```diff
@@ -20,19 +21,29 @@ export class LegendaryUser {
     return new Promise((res) => {
       const child = spawn(legendaryBin, command)
       child.stderr.on('data', (data) => {
-        console.log(`stderr: ${data}`)
+        logInfo(`stderr: ${data}`)
         if (`${data}`.includes('ERROR')) {
+          logError(`${data}`)
+          dialog.showErrorBox(
+            i18next.t('box.error.login.tittle', 'Cannot Login!'),
+            `${data}`
+          )
           return res('error')
         }
       })
       child.stdout.on('data', (data) => {
-        console.log(`stderr: ${data}`)
+        logInfo(`stderr: ${data}`)
         if (`${data}`.includes('ERROR')) {
+          logError(`${data}`)
+          dialog.showErrorBox(
+            i18next.t('box.error.login.tittle', 'Cannot Login!'),
+            `${data}`
+          )
           return res('error')
         }
```

Comment:
Same as above. Also in my eyes stderr should be stdout in logInfo.

```suggestion
        if (`${data}`.includes('ERROR')) {
          logError(`${data}`, LogPrefix.Legendary)
          dialog.showErrorBox(
            i18next.t('box.error.login.tittle', 'Cannot Login!'),
            `${data}`
          )
          return res('error')
       } else {
          logInfo(`stdout: ${data}`, LogPrefix.Legendary)
       }
```